universal-application-tool-0.0.1/app/views/questiontypes/ApplicantQuestionRendererFactory.java

```diff
@@ -12,10 +13,18 @@
 import services.question.types.QuestionDefinition;
 import services.question.types.QuestionDefinitionBuilder;
 import services.question.types.QuestionType;
+import views.FileUploadViewStrategy;
 
 /** A helper class for constructing type-specific applicant question renderers. */
 public class ApplicantQuestionRendererFactory {
 
+  private final FileUploadViewStrategy fileUploadViewStrategy;
+
+  @Inject
```

Comment:
issue (blocking): adding `@Inject` indicates `ApplicantQuestionRendererFactory` should be initialized and injected by guice, but I'm seeing it initialized manually in several places. It should consistently be one or the other.

Reply:
Similarly I believe @Inject methods shouldn't be public and should have the default package exposure.

Reply:
I see! I think I was misunderstanding how constructor injection works. I updated so that the only injected constructors are the ones that already existed before this PR.